iOS/GPUImage/component/component.yaml

```diff
@@ -13,7 +14,18 @@ libraries:
 samples:
   - name: "GPUImage Sample"
     path:  ../samples/GPUImageSample.sln
+    removeProjects:
+        - GPUImage
+    installNuGets:
+      - project: GPUImageSample
+        packages:
+          - Xamarin.GPUImage
 no_build: true
 is_shell: true
 packages:
-  ios-unified: GPUImage, Version=0.1.7.1
\ No newline at end of file
+  ios-unified: GPUImage, Version=0.1.7.
```

Comment:
there is an issues with this version number.  Its missing the last digit of the version number